docs/source/dashboard.rst

```diff
@@ -0,0 +1,250 @@
+   client = Client()  # start distributed scheduler locally. 
+   client            
+
+In a Jupyter Notebook or Jupyter Lab session displaying the client object will show the dashboard address
+as following.
+
+.. figure:: images/dashboard_link.png
+    :alt: Client html repr displaying the dashboard link on a JupyterLab session. 
+
+The address of the dashboard will be displayed if you are in a Jupyter Notebook,
+or, if you are in a terminal or IPython, it can be queried from ``client.dashboard_link``. By default, when starting a scheduler 
+n your local machine the dashboard will be served at ``http://localhost:8787/status``, you
+can type this address into your browser to access the dashboard but may be served 
+elsewhere if port 8787 is taken. You can also configure the address by passing options to the 
+scheduler, see ``dashboard_address`` in `LocalCluster <https://docs.dask.org/en/stable/deploying-python.html#reference>`__
+
+The dashboard link redirects you to main dashboard page as shown below:
+
+.. figure:: images/dashboard_status.png
+    :alt: Local host dashboard status page displaying the five panes that compose it. The Byte stored pane showing blue horizontal bar of 
+    the cluster memory stored up to 593.80 MiB, the Bytes stored per worker pane with four blue horizontal bars of approximately 150 MiB
+    each, the Task Processing pane with four blue bars up to 350 tasks per worker, the Task Stream pane showing eight horizontal bars with 
+    different shades of greens and blues and some red, and the Progress pane showing horizontal bars for each task-prefix matching the 
+    blue and green colors of the task stream pane. 
+
+In the entry point of the dashboard you can find multiple plots with information about your cluster 
+as listed below:
+
+- :ref:`dashboard.memory`: Cluster memory and Memory per worker
+- :ref:`dashboard.proc-cpu-occ`:  Tasks being processed by each worker/ CPU Utilization per worker/ Expected runtime
+for all tasks currently on a worker.
+- :ref:`dashboard.task-stream`: Individual task across threads.
+- :ref:`dashboard.progress`: Progress of a set of tasks.
+
+.. _dashboard.memory: 
+
+Bytes Stored and Bytes per Worker
+---------------------------------
+These two plots show a summary of the overall memory usage on the cluster (Bytes Stored),
+as well as the individual usage on each worker (Bytes per Worker). The colors on these plots 
+indicate the following.  
+
+.. figure:: images/dashboard_memory.png
+    :alt: The Byte stored pane showing blue and grey horizontal bar of the cluster memory, the blue portion 
+    corresponds to 12.6 GiB of memory while the grey portion represents the 750 MiB spilled to disc. The Bytes 
+    stored per worker pane showing 16 workers each of them with a horizontal bar. The first one from the top is 
+    orange and grey, the following three are orange, and the remaining twelve are blue. The orange and blue bars 
+    also have different levels of transparency. 
```

Comment:
This suggestion I think it's a bit confusing because we don't explain in this document what are managed and unmanaged memory. I don't have a better solution yet, but I'll think about it.